heuristic/delta_calculations.py

```diff
@@ -1,43 +1,29 @@
+from operator import itemgetter
 from collections import defaultdict
 from copy import copy
-from operator import itemgetter
-
+def delta_calculate_deviation_from_demand(state, competencies, t_covered_by_shift, employee_with_competencies, demand, destroy_repair_set):
```

Comment:
Is there a particular reason for why this is not a class while `Alns_calculator` is?

Reply:
ALNS_calculator was my first implementation. I felt it was not necessary to create a class for only functions. I feel this structure is better. I am not a fan of making classes just to make them.

Reply:
ALNS_calculator is also outdated

Reply:
Roger!